UnityProject/Assets/Scripts/Health/Operable.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace Health
+{
+    /// <summary>
+    /// This component allows for a living object to receive surgery
+    /// </summary>
+    [RequireComponent(typeof(HealthSystem))]
+    public class Operable : MonoBehaviour, ICheckedInteractable<HandApply>
+    {
+        protected static readonly StandardProgressActionConfig ProgressConfig
+            = new StandardProgressActionConfig(StandardProgressActionType.Restrain);
+    
+        private RegisterTile registerTile = null;
+        //The initial surgery initiation page
+        public NetTabType NetTabType = NetTabType.Surgery;
```

Comment:
I think I already said it in the outdated comment, but I say it again. If this is for the inspector, use Tooltip instead of regular comments